x/ccv/provider/client/cli/tx.go

```diff
@@ -101,11 +102,18 @@ Example:
 			txf = txf.WithTxConfig(clientCtx.TxConfig).WithAccountRetriever(clientCtx.AccountRetriever)
 
 			submitter := clientCtx.GetFromAddress()
-			var misbehaviour ibctmtypes.Misbehaviour
-			if err := clientCtx.Codec.UnmarshalInterfaceJSON([]byte(args[1]), &misbehaviour); err != nil {
+			misbRaw, err := os.ReadFile(args[0])
```

Comment:
Maybe rename `misbRaw` to `misbehaviourJson` similarly to what is being done in double-voting evidence with:
```
evidenceJson, err := os.ReadFile(args[0])
```
?